### DISCUSSION TITLE
How to scan using nuclei

### DISCUSSION THREAD

**Discussion by sudip444** [projectdiscovery/nuclei#1998] [General]
### Current Behavior:when try to scan a site or subdomain list,it always  shows no results found, better luck next time
.
### Steps To Reproduce:
Example: steps to reproduce the behavior:
1. Run 'nuclei -t /root/nuclei-templates/cves/ -l test.txt
2. See no result found, better luck next time. 
-->

**Comment by ehsandeep** [comment]
@sudip444 nuclei accept root URLs as input and not subdomains, see the example input: https://github.com/projectdiscovery/nuclei#running-nuclei

**Reply by sudip444** [reply]
Also when I try to scan a domain it shows the same.

**Reply by ehsandeep** [reply]
Do you have an example of sharing what you included in `l.txt`? also, since you are scanning for cves, you will only get results if the targets are vulnerable.
